index.html

```diff
@@ -1,15 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <!-- HTML:
-    form - Select Input(Different Alcohols)
-    submit button - Give Me My Cocktail -->
-
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>What Cocktail Should I Make?</title>
+        <link rel="stylesheet" href="styles/styles.css" />
+    </head>
+    <body>
+        <main>
+            <header>
+                <h1>What Cocktail Should I Make?</h1>
+                <img
+                    src="https://images.unsplash.com/photo-1611873027887-19a588803816?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=627&q=80"
+                    alt="#"
+                />
+            </header>
+
```

Comment:
For styling purposes in our design I think the form element should be in our header. 

The results and subsequent buttons could be our main content